src/shared/sleep-config.c

```diff
@@ -260,15 +271,43 @@ static bool enough_memory_for_hibernation(void) {
         return r;
 }
 
+static bool can_s2h(void) {
+        int r;
+
+        r = access("/sys/class/rtc/rtc0/wakealarm", W_OK);
+        if (r < 0) {
+                log_error_errno(r, "Unable to write to RTC.");
```

Comment:
This should not be `log_error`, because that'd mean an error would be emitted on all systems without an rtc0. See how this is done above with /proc/swaps (https://github.com/systemd/systemd/pull/8274/files#diff-897a4693906dd8751a19d0c70f1f4207L188) — -ENOENT results in log_debug, and other errors in a warning.

Also "Unable to write to RTC" is misleading, because it suggests that we attempted to write, and couldn't. Something like "/sys/class/rtc/rtc0/wakealarm is not writable" would be better.